yb-voyager/cmd/common.go

```diff
@@ -206,9 +206,13 @@ func displayExportedRowCountSnapshotAndChanges() {
 			uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"), headerfmt("TOTAL CHANGES EVENTS"),
 				headerfmt("INSERTS"), headerfmt("UPDATES"), headerfmt("DELETES"),
 				headerfmt("FINAL ROW COUNT(SNAPSHOT + CHANGES)"))
-
 		}
-		totalChangesEvents, inserts, updates, deletes, err := metaDB.GetExportedEventsStatsForTable(tableStatus.SchemaName, tableStatus.TableName)
+		schemaName := tableStatus.SchemaName
+		sourceSchemaCount := len(strings.Split(source.Schema, "|"))
+		if sourceSchemaCount <= 1 {
+			schemaName = ""
+		}
+		totalChangesEvents, inserts, updates, deletes, err := metaDB.GetExportedEventsStatsForTable(schemaName, tableStatus.TableName)
```

Comment:
shall we hide the `schemaName` column from UI table.
My concern is tomorrow some people might start saying that why it is empty, please fill with the correct information.

Reply:
Ah, missed this! Let me incorporate it.